config/vsphere-csi-driver/overwrites/values.yaml

```diff
@@ -54,8 +19,14 @@ storageClass:
     vcdStorageProfileName: "vSAN Default Storage Policy"
     fileSystem: "ext4"
 
+internalFeatureStates:
+  topologyPreferentialDatastores:
+    enabled: false
+
 containerSecurityContext:
   allowPrivilegeEscalation: false
   runAsNonRoot: true
+  runAsUser: 65532
```

Comment:
I introduced this here because someone introduced a bug upstream by not updating all the containers with this user ID. Until this is fixed upstream, this workaround works.